sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -959,6 +959,11 @@ object Radar {
 
         val googlePlayProjectNumber = RadarSettings.getGooglePlayProjectNumber(this.context);
 
+        if (googlePlayProjectNumber == 0.toLong()) {
+            callback?.onComplete(RadarStatus.ERROR_SERVER)
```

Comment:
is `ERROR_SERVER` the right status to return here? feels more like a misconfiguration than us encountering an error

Reply:
Yes, it could be a misconfig, though in practice we've experienced it from `getConfig` failing.

What other error code do you suggest?

Reply:
going with `ERROR_BAD_REQUEST`